bundles/org.openhab.binding.flume/src/main/java/org/openhab/binding/flume/internal/api/dto/FlumeApiQueryWaterUsage.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.flume.internal.api.dto;
+
+import java.time.LocalDateTime;
+
+import org.openhab.binding.flume.internal.api.FlumeApi;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link FlumeApiQueryWaterUsage} dto for setting up query of water usage.
+ *
+ * @author Jeff James - Initial contribution
+ */
+public class FlumeApiQueryWaterUsage {
+    @SerializedName("request_id")
+    public String requestId;
+    @SerializedName("since_datetime")
+    public LocalDateTime sinceDateTime;
+    @SerializedName("until_datetime")
+    public LocalDateTime untilDateTime;
+    public String tz;
```

Comment:
Minor, but i would prefer a more meaningfull: `timeZone` 
```suggestion
    @SerializedName("tz")
    public String timeZone;
```